doit/task.py

```diff
@@ -165,7 +164,7 @@ def __init__(self, name, actions, file_dep=(), targets=(),
                  subtask_of=None, has_subtask=False,
                  doc=None, params=(), pos_arg=None,
                  verbosity=None, title=None, getargs=None,
-                 watch=(), loader=None, cfg_defaults=dict()):
+                 watch=(), loader=None, cfg_values=None):
```

Comment:
better remove from `__init__` parameters. It is not expected to set this value on Task creation.

Reply:
Fixed.